py/bc0.h

```diff
@@ -101,12 +101,12 @@
 #define MP_BC_ROT_TWO                       (MP_BC_BASE_BYTE_O + 0x0a)
 #define MP_BC_ROT_THREE                     (MP_BC_BASE_BYTE_O + 0x0b)
 
+#define MP_BC_UNWIND_JUMP                   (MP_BC_BASE_JUMP_E + 0x00) // rel byte code offset, 16-bit signed, in excess; then a byte
```

Comment:
What does "in excess" mean in this context?

Reply:
- if the high bit of the first byte is clear, then it's in excess by 0x40, ie the value is `ip[0] - 0x40`
- if the high bit is set, then it's 2 bytes in excess by 0x4000, ie the value is `(ip[0] & 0x7f | ip[1] << 7) - 0x4000`

Maybe I should write such a definition somewhere in this file and refer to it in the comment.

Reply:
> Maybe I should write such a definition somewhere in this file and refer to it in the comment.

Yes, this would be helpful to the uninitiated. :smile:

Reply:
I have added and updated comments.